node/src/components/rest_server.rs

```diff
@@ -0,0 +1,149 @@
+//! API server
```

Comment:
Similar to my last comment - I think these docs are out of date now that we are `rest_server.rs`

Reply:
Did this.